rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -359,6 +360,13 @@ upgrade_ncf() {
 
 }
 
+# Upgrade masterfiles
+upgrade_masterfiles() {
+  # - 3.0.13  Now we should have a masterfiles initialized from initial promises
+  STEP="Upgrade masterfiles from initial promises"
+  rsync --delete -rptgoq "${RUDDER_SHARE}/initial-promises/" "${RUDDER_VAR}/cfengine-community/masterfiles/"
```

Comment:
Actually, re-reading the comments (in particular: "if the agent is not installed, the initial-promises aren't there"), we need a different protection on this: test that ```"${RUDDER_SHARE}/initial-promises/"``` exists.